src/worldgen/DensityFunction.ts

```diff
@@ -697,12 +715,12 @@ export namespace DensityFunction {
 					max = max1 + max2
 					break
 				case 'mul':
-					min = min1 > 0 && min2 > 0 ? min1 * min2
-						: max1 < 0 && max2 < 0 ? max1 * max2
-							: Math.min(min1 * max2, min2 * max1)
-					max = min1 > 0 && min2 > 0 ? max1 * max2
-						: max1 < 0 && max2 < 0 ? min1 * min2
-							: Math.max(min1 * min2, max1 * max2)
+					min = min1 > 0 && min2 > 0 ? (min1 * min2) || 0
+						: max1 < 0 && max2 < 0 ? (max1 * max2) || 0
+							: Math.min((min1 * max2) || 0, (min2 * max1) || 0)
+					max = min1 > 0 && min2 > 0 ? (max1 * max2) || 0
+						: max1 < 0 && max2 < 0 ? (min1 * min2) || 0
+							: Math.max((min1 * min2) || 0, (max1 * max2) || 0)
```

Comment:
`Infinity * 0 = NaN`. usually we want the result of that multiplication to be 0 though. I think this "bug" exists in vanilla Minecraft as well, and it doesn't hurt anything except performance, but having NaN as the min and max values everywhere is ugly...